Fix Twitter_Bootstrap_Form if getView is NULL

dready opened the pull request:
hi,

in Twitter_Bootstrap_Form::_initializePrefixes adds an HelperPath to the View. Unfortunately if you are using Unittests on the Form there is no View. Therefore i added an is_null check.

best regards, dready

Commit:
check if View is null

check if view is null before using it. This happens usually in unittests.

theUniC commented:
This PR has two fixes, for two different things. Can you split them in two separate PR, please?

dready commented:
.. damn! sure. was not supposed to do that^^

dready commented:
done.

Commit:
adjust condition and fix getView

dready commented:
fixed getView is a method and changed the comparison.

But i don"t think it should join the other conditional since it"s only checking if getView() returns a valid view. This has nothing to do with the Prefixes.
